chasm/field_test.go

```diff
@@ -12,7 +12,6 @@ import (
 	"go.temporal.io/server/common/log"
 	"go.temporal.io/server/common/testing/protorequire"
 	"go.temporal.io/server/common/testing/testlogger"
-	"go.temporal.io/server/common/testing/testvars"
```

Comment:
Why you cannot use `testvars` here?

Reply:
it depends on tons of packages to return typed values. one of them is "go.temporal.io/server/common/worker_versioning" which also some how depends on persistence...

```
go: can't load test package: package go.temporal.io/server/chasm
        imports go.temporal.io/server/common/testing/testvars from tree_test.go
        imports go.temporal.io/server/common/worker_versioning from test_vars.go
        imports go.temporal.io/server/common/persistence/visibility/manager from worker_versioning.go
        imports go.temporal.io/server/common/persistence from visibility_manager.go
        imports go.temporal.io/server/chasm from data_interfaces.go: import cycle not allowed in test
```